### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 17

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#289] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152319751-7e9d8df2-ab4a-44f4-9626-e7d22abebde9.png)

**Comment by LuginiAndrea** [comment]
Vediamo subito che non è transitiva: $4 R^{\cup} 2, 2 R^{\cup} 1, \nexists 4 R^{\cup} 1$
$S \circ S = a R^{\circ }c \Longleftrightarrow aRb \vee bRc \Longrightarrow \left(2,5\right)$
R è già la chiusura transitivà di se stessa, essendo già transitiva (notare che non sussite mai il caso di non transitività, quindi è transitiva)
